helloworld/util/backend.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from helloworld.util.mode import Mode, get_mode
+
+
+def get_backend_url(name: str) -> str:
+    if get_mode() == Mode.DEV:
+        from django.conf import settings
```

Comment:
also importing get_modes already causes `from django.conf import settings` to run... so this import, if needed can be at the top of the file.